pkg/storage/stores/shipper/util/util.go

```diff
@@ -0,0 +1,68 @@
+package util
```

Comment:
A bit curious why we created a new package for this -- any thoughts about putting this in the `shipper` pkg?

Reply:
I have moved common code to it. If I move it to `shipper` package then there would be circular dependency since that common code is being used by `downloads` package which is being imported by `shipper` package.